core/test/lib/lantern/metrics/speed-index-test.js

```diff
@@ -10,19 +10,20 @@ import {readJson} from '../../../test-utils.js';
 import {SpeedIndex} from '../../../../lib/lantern/metrics/speed-index.js';
 import {FirstContentfulPaint} from '../../../../lib/lantern/metrics/first-contentful-paint.js';
 import {getComputationDataFromFixture} from './metric-test-utils.js';
-import {Speedline} from '../../../../computed/speedline.js';
 
 const trace = readJson('../../../fixtures/artifacts/progressive-app/trace.json', import.meta);
 
 const defaultThrottling = constants.throttling.mobileSlow4G;
 
 describe('Metrics: Lantern Speed Index', () => {
   it('should compute predicted value', async () => {
-    const context = {computedCache: new Map()};
     const data = await getComputationDataFromFixture({trace});
+    // TODO: observedSpeedIndex is from the Speedline library, and is used for optimistic
+    // mode. At the moment callers must pass the result into Lantern.
```

Comment:
nit: what is the TODO here? Eventually move speedline inside the lantern library if we need it?